.github/workflows/cypress-tests.yml

```
@@ -20,5 +20,4 @@ jobs:
     - name: Cypress run
       uses: cypress-io/github-action@v6
       with: 
-        build: npm run build
-        start: npm start
+        start: npm run test:Component
```

```suggestion
        start: npm run test:Component
        component: true
```
Maybe we need to change the testing mode in the action? https://github.com/cypress-io/github-action?tab=readme-ov-file#component-testing

---

I made this change and run `act` and the workflow succeeded.